packages/node/src/store.ts

```diff
@@ -206,6 +206,10 @@ export class Store {
   public async addVirtualAppInstanceProposal(
     proposedAppInstance: ProposedAppInstanceInfo
   ) {
+    const sortedXpubs = [
```

Comment:
Why not use `sortAddresses` from xkeys.ts?

Reply:
Because these are the sorted xpubs not the sorted addresses